packages/@webex/internal-plugin-metrics/src/call-diagnostic/call-diagnostic-metrics.ts

```diff
@@ -160,6 +164,7 @@ export default class CallDiagnosticMetrics extends StatelessWebexPlugin {
           os: getOSNameInternal(),
           browser: getBrowserName(),
           browserVersion: getBrowserVersion(),
+          clientLaunchMethod: options?.clientLaunchMethod, // do we want something by default
```

Comment:
i think we should do it at the end of the function, 
if options. clientLaunchMethod {
  origin. clientLaunchMethod = ...
}

Reply:
But clientLaunchMethod is at origin.clientInfo.clientLaunchMethod level so I think you mean at end of the function set it at that level

Reply:
yes sorry